src/sysfiles.c

```diff
@@ -711,41 +774,67 @@ const Char * SyWinCmd (
 */
 
 
-/****************************************************************************
-**
-*V  syBuf . . . . . . . . . . . . . .  buffer and other info for files, local
-**
-**  'syBuf' is  a array used as  buffers for  file I/O to   prevent the C I/O
-**  routines  from   allocating their  buffers  using  'malloc',  which would
-**  otherwise confuse Gasman.
-**
-**
-**  Actually these days SyBuf just stores various file info. SyBuffers
-**  stores buffers for the relatively few files that need them.
-*/
-
-SYS_SY_BUF syBuf [256];
-
-SYS_SY_BUFFER syBuffers [ 32];
-
 // Mark a member of syBuf as unused
-void SyMarkBufUnused(Int i)
+static void SyBufMarkUnused(Int i)
 {
     GAP_ASSERT(i >= 0 && i < 256);
     memset(&(syBuf[i]), 0, sizeof(syBuf[i]));
     syBuf[i].type = unused_socket;
+    syBuf[i].fp = -1;
 }
 
 // There is no explicit method to mark syBufs as used,
 // they are marked as used when created.
 
 // Check if a member of syBuf is in use
-Int SyBufInUse(Int i)
+static Int SyBufInUse(Int i)
 {
     GAP_ASSERT(i >= 0 && i < 256);
     return syBuf[i].type != unused_socket;
 }
 
+void SyRedirectStderrToStdOut(void)
```

Comment:
Hmm, `Stderr` vs. `StdOut`... should unify the capitalization... Also, strictly speaking errin/errout are mapped to stdin/stdout, so the name is not quite correct; if anybody has a better idea, let me know!